README.md

```diff
@@ -7,37 +7,33 @@
 This repo contains our mirror landing page.
 https://pkg.adfinis.com
 
-The page is compiled and minified using nodejs broccoli and then uploaded to our mirror.
+The page is built with vite and then uploaded to our mirror.
 
 ## Requirements
 
 NodeJS and yarn must be installed on your system.
 
-broccoli and its dependencies are installed using yarn:
+The dependencies of the project can be installed using yarn:
```

Comment:
```suggestion
The dependencies of the project need to be installed using yarn:
```

Reply:
also, it does this on a fresh clone of this branch:
```console
$ yarn
yarn install v1.22.19
[1/4] Resolving packages...
[2/4] Fetching packages...
warning solid-markdown@2.0.0: The engine "pnpm" appears to be invalid.
[3/4] Linking dependencies...
warning " > @babel/eslint-parser@7.23.3" has unmet peer dependency "@babel/core@^7.11.0".
warning "eslint-plugin-solid > @typescript-eslint/utils > @typescript-eslint/typescript-estree > ts-api-utils@1.0.3" has unmet peer dependency "typescript@>=4.2.0".
warning " > vite-plugin-fonts@0.7.0" has incorrect peer dependency "vite@^2.0.0 || ^3.0.0 || ^4.0.0".
error An unexpected error occurred: "EEXIST: file already exists, mkdir '/home/hairmare/Documents/git.repos/adsy/landingpage-repo_mirror/node_modules'".
info If you think this is a bug, please open a bug report with the information provided in "/home/hairmare/Documents/git.repos/adsy/landingpage-repo_mirror/yarn-error.log".
info Visit https://yarnpkg.com/en/docs/cli/install for documentation about this command.
```

Reply:
should be fixed (workaround) here https://github.com/adfinis/landingpage-repo_mirror/pull/24/commits/b1213e572fbb61be58ed3d95991de80ff35d7d9c